include/wx/osx/private/uilocale.h

```diff
@@ -0,0 +1,23 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/osx/private/uilocale.h
+// Purpose:     Helper for making pointer to current NSLocale available
+//              for locale-dependent controls under macOS.
+// Author:      Vadim Zeitlin
+// Created:     2023-10-13
+// Copyright:   (c) 2023 Vadim Zeitlin <vadim@wxwidgets.org>
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_OSX_PRIVATE_UILOCALE_H_
+#define _WX_OSX_PRIVATE_UILOCALE_H_
+
+#if wxUSE_INTL
+
+#ifdef __WXOSX_COCOA__
```

Comment:
This is not needed here, is it?

Reply:
No, probably not. At least if that header is not included from some non-Cocoa build. I don't know whether the latter can happen or not.

Reply:
I'd still remove it.

Reply:
Removed.